src/providers/twitch/TwitchChannel.cpp

```diff
@@ -85,7 +85,8 @@ TwitchChannel::TwitchChannel(const QString &name,
     , ChannelChatters(*static_cast<Channel *>(this))
     , subscriptionUrl_("https://www.twitch.tv/subs/" + name)
     , channelUrl_("https://twitch.tv/" + name)
-    , popoutPlayerUrl_("https://player.twitch.tv/?channel=" + name)
+    , popoutPlayerUrl_("https://player.twitch.tv/?channel=" + name +
+                       "&parent=twitch.tv")
```

Comment:
@alazymeme I think that format `?parent=twitch.tv&channel=" + name` will be a bit better, because it's in a one string only.

Reply:
> @alazymeme I think that format `?parent=twitch.tv&channel=" + name` will be a bit better, because it's in a one string only.

According to twitch's dev docs, the correct usage is `?channel=channel&parent=sitethatwillembedthecontent`. However twitch re-organizes them anyway. So I'm not sure if it NEEDS to be one way or the other...